lib/Net/ThreeScale/Client.pm

```diff
@@ -48,7 +48,7 @@ sub new {
 
 	my $self = {};
 	$self->{provider_key} = $params->{provider_key};
-	$self->{url}          = $params->{url} || 'http://su1.3scale.net';
+	$self->{url}          = $params->{url} || 'https://su1.3scale.net';
```

Comment:
Https is fine, but requires a client that keeps persistent connection and keep-alive. Otherwise the performance is atrocious. Before switching the default we would need to be positive it is using keepalive IMO.

But maybe there is not so many users and the impact will be minimal. How it worked for you?